bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -226,49 +170,33 @@ Number Netatmo_Outdoor_Temperature "Temperature" { channel = "netatmo:NAModule1:
-| DateMaxHumidityThisMonth| DateTime         | Date when maximum humidity was reached this month        |
-| Humidex             | Number               | Computed Humidex index                                   |
-| HeatIndex           | Number:Temperature   | Computed Heat Index                                      |
+| humidex             | Number               | Computed Humidex index                                   |
+| heat-index           | Number:Temperature   | Computed Heat Index                                      |
 | Dewpoint            | Number:Temperature   | Computed dewpoint temperature                            |
-| DewpointDepression  | Number:Temperature   | Computed dewpoint depression                             |
-| MinTemp             | Number:Temperature   | Minimum temperature on current day                       |
-| MinTempThisWeek     | Number:Temperature   | Minimum temperature this week                            |
-| MinTempThisMonth    | Number:Temperature   | Minimum temperature this month                           |
-| MaxTemp             | Number:Temperature   | Maximum temperature on current day                       |
-| MaxTempThisWeek     | Number:Temperature   | Maximum temperature this week                            |
-| MaxTempThisMonth    | Number:Temperature   | Maximum temperature this month                           |
+|dew-point-depression  | Number:Temperature   | Computed dewpoint depression                             |
+| today-min             | Number:Temperature   | Minimum temperature on current day                       |
+| today-max             | Number:Temperature   | Maximum temperature on current day                       |
 | DateMinTemp         | DateTime             | Date when minimum temperature was reached on current day |
-| DateMinTempThisWeek | DateTime             | Date when minimum temperature was reached this week      |
-| DateMinTempThisMonth| DateTime             | Date when minimum temperature was reached this month     |
 | DateMaxTemp         | DateTime             | Date when maximum temperature was reached on current day |
-| DateMaxTempThisWeek | DateTime             | Date when maximum temperature was reached this week      |
-| DateMaxTempThisMonth| DateTime             | Date when maximum temperature was reached this month     |
-| TimeStamp           | DateTime             | Timestamp when data was measured                         |
-| LastMessage         | DateTime             | Last message emitted by the module                       |
+| timestamp           | DateTime             | Timestamp when data was measured                         |
+| last-message         | DateTime             | Last message emitted by the module                       |
 | LowBattery          | Switch               | Low battery                                              |
```

Comment:
Not sure, but it seems this should be **battery#low-battery** now for all modules?